src/style.css

```
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
```

```suggestion
.stars--1 .stars__star:first-child {
  background-image: url(./images/star-active.svg);
}
```
you dont need this everywhere